### PR TITLE
add the 3 new chip classes: Date, Person, and RichLink

### PR DESCRIPTION
**Pull request by zevdg**
See https://developers.google.cn/apps-script/releases?hl=fr-SN#august_23_2021

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.cn/apps-script/releases?hl=fr-SN#august_23_2021
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/google-apps-script/google-apps-script.document.d.ts [hunk 1]
```diff
@@ -1462,6 +1465,53 @@ declare namespace GoogleAppsScript {
      *     table.getCell(0, 2).setVerticalAlignment(DocumentApp.VerticalAlignment.BOTTOM);
      */
     enum VerticalAlignment { BOTTOM, CENTER, TOP }
+
+    interface Date extends Element {
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.document.d.ts:1469]: Move between `ContainerElement` and `Document` (sort alphabetically).
Person, RickLink also.

File: types/google-apps-script/google-apps-script.document.d.ts [hunk 2]
```diff
@@ -1462,6 +1465,53 @@ declare namespace GoogleAppsScript {
...
+      getLocale(): string;
+      getNextSibling(): Element;
+      getParent(): ContainerElement;
+      getPreviousSibling(): Element;
+      getTimestamp(): Date;
+      getType(): ElementType;
+      isAtDocumentEnd(): boolean;
+      merge(): Date;
+      removeFromParent(): Date;
+      setAttributes(attributes: any): Date;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.document.d.ts:1482]: It is better to define type of arg attributes.

```suggestion
      setAttributes(attributes: {[key in DocumentApp.Attribute]: any}): Date;
```

```ts
const foo = (attributes: {[key in Attribute]: any}) => {
    attributes[DocumentApp.Attribute.BACKGROUND_COLOR] = 1; // ok
    attributes["bar"] = 1; // compile error
}
```

**Review by mtgto — CHANGES_REQUESTED** [review]

File: types/google-apps-script/google-apps-script.document.d.ts (hunk 2 above)
Comment (zevdg) [reply types/google-apps-script/google-apps-script.document.d.ts:1482]: Hmmm, there are lots of other uses of attribute spread throughout the rest of this typing and they all use `any`.  Shouldn't they be consistent?  Someone could switch all of them, but I don't know enough to be confident that switching them all wouldn't break existing code.  Even if I did, I'd still opt to make that change in a separate PR.

File: types/google-apps-script/google-apps-script.document.d.ts (hunk 1 above)
Comment (zevdg) [reply types/google-apps-script/google-apps-script.document.d.ts]: Done.

Comment (zevdg) [reply types/google-apps-script/google-apps-script.document.d.ts]: also added some comments.

File: types/google-apps-script/google-apps-script.document.d.ts (hunk 2 above)
Comment (mtgto) [reply types/google-apps-script/google-apps-script.document.d.ts]: Okay, leave this issue as it is. I welcome if you make a another PR to refactor it, but it is not required 😃

**Review by mtgto — APPROVED** [review]
LGTM. Thank you for your contribution 👍

**Pull request merged**